docker-compose.yml

```diff
@@ -3,14 +3,14 @@ services:
   config-server:
     image: mszarlinski/spring-petclinic-config-server
     container_name: config-server
-    mem_limit: 256M
+    mem_limit: 1024M
```

Comment:
What is the reason to increase the memory limit from 256M to 1024M ?

Reply:
Yes, it works and i have changed the pull request accordingly.